lib/wprof/conf_wprof.rb

```diff
@@ -0,0 +1,25 @@
+module WProf
+  class Config
+    DEFAULTS_CONFIGS = {
+      db_runtime: true,
+      reporter_type: 'LOGGER',
+      csv_type: 'SPLIT',
+      async: false,
+      httparty_methods_to_trace: %w[get put delete post patch].freeze,
+      external_url: nil,
+      external_headers: nil,
+      custom_methods: nil,
+      file_path: Rails.root.to_s
+    }.freeze
+
+    class << self
+      def get_value(param)
+        unless Rails.application.nil?
+          value = Rails.configuration.x.wprof[param]
+          return value unless value.nil?
```

Comment:
```suggestion
          return value if value.present?
```

Reply:
This check it's kind of weird, why don't you simply call `Rails.configuration.x.wprof[param]`? I think it behaves the same as what you are doing right now

Reply:
x.wprof only exist if you defined custom configuration. If you don't do it, config must be return custom values. I can't do 
``` ruby
return Rails.configuration.x.wprof[param] if Rails.configuration.x.wprof[param].present? 
```
because, it's too long and ugly.